lib/cavendish/assets/src/providers/AppProvider.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { TailwindProvider, Utilities } from 'tailwind-rn';
+
+import utilities from '../../tailwind.json';
+
+export default function AppProvider({ children }: React.PropsWithChildren) {
+  return (
+    <TailwindProvider utilities={utilities as Utilities}>
+      {children}
+    </TailwindProvider>
+  ); 
+}
```

Comment:
You're missing a newline at the end of this file

Reply:
thanks!